content/3_Silver/Flood_Fill.mdx

```diff
@@ -739,3 +739,66 @@ print(room_num)
 ## Problems
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+	<Quiz.Question>
+		How many times will we have to run DFS to perform a flood fill on a graph G with 4 connected components?
+		<Quiz.Answer>
+			1
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require $\mathbf{4}$ DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			2
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			3
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			4
+			<Quiz.Explanation>
+				Correct! Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+	</Quiz.Question>
+
+	<Quiz.Question>
+		Which of the following is not information we could get from the results of a floodfill algorithm?
+		<Quiz.Answer>
+			The number of connected components of a graph
+			<Quiz.Explanation>
+				Incorrect. We find the number of connected components by tracking the number of times DFS is executed.
+			</Quiz.Explanation>
+		</Quiz.Answer> 
+		<Quiz.Answer>
+			The number of nodes in each connected component
+			<Quiz.Explanation>
+				Incorrect. We can find the number of nodes in each connected component by iterating through all nodes after running floodfill and determining which connected component each node is in.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			The shortest path between two nodes
+			<Quiz.Explanation>
+				Correct! This cannot be found with floodfill, because floodfill doesn't keep track of distances.
```

Comment:
not sure how I feel about this :|

Reply:
I think this question is okay. You can use BFS to flood fill and maintain shortest distances at the same time, but we haven't described how to maintain shortest distances in this module.

Reply:
i could add "with a standard floodfill"